packages/rest/README.md

```diff
@@ -52,16 +52,37 @@ Application options.
 const app = new RestApplication({
   rest: {
     port: 3001,
+
+    // Template of the OpenAPI spec, which will be filled with `paths` from
+    // controllers.
+    openApiSpec: {
+      info: {
+        title: 'My Application',
+        version: '1.0.0',
+      },
+      servers: [{url: 'http://127.0.0.1:8080'}],
+    },
+
+    // Optional configuration for API explorer
+    apiExplorer: {
+      // URL for the hosted API Explorer UI
+      // default to https://loopback.io/api-explorer
+      explorerUrl: 'http://petstore.swagger.io',
+      // Set `servers` based on HTTP request headers, default to `false`
+      setServersFromRequest: false,
+    },
   },
 });
 ```
 
 ### `rest` options
 
-| Property | Type            | Purpose                                                                                                   |
-| -------- | --------------- | --------------------------------------------------------------------------------------------------------- |
-| port     | number          | Specify the port on which the RestServer will listen for traffic.                                         |
-| sequence | SequenceHandler | Use a custom SequenceHandler to change the behavior of the RestServer for the request-response lifecycle. |
+| Property    | Type               | Purpose                                                                                                   |
+| ----------- | ------------------ | --------------------------------------------------------------------------------------------------------- |
+| port        | number             | Specify the port on which the RestServer will listen for traffic.                                         |
+| sequence    | SequenceHandler    | Use a custom SequenceHandler to change the behavior of the RestServer for the request-response lifecycle. |
+| openApiSpec | OpenApiSpec        | Base template of the OpenAPI spec                                                                         |
```

Comment:
This is not entirely correct. `OpenApiSpec` type requires `openapi` version and `paths` object to be present.